docs/source/listings.rst

```diff
@@ -220,3 +220,45 @@ Cloud Storage Buckets
         including image cutouts and metadata. Each alert is stored as a separate Avro file.
         The filename syntax is: `<schema_version>/<alert_date>/<diaObjectId>/<diaSourceId>.avro`.
         For example, `v7_3/2026-10-01/3516505565058564097/3527242976319242284.avro`.
+
+.. _data swift:
+
+Swift/BAT-GUANO
+-------------------------------
+
+:ref:`Neil Gehrels Swift Observatory <survey swift>` is a space-based observatory designed to study gamma-ray bursts and currently produces an alert stream at a rate of less than 1 alert per week.
+
+Pub/Sub Alert Streams
+^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table::
+    :class: tight-table
+    :widths: 25 75
+    :header-rows: 1
+
+    * - Topic
+      - Description
+
+    * - .. centered:: *Data Streams*
+      -
+
+    * - swift-alerts
+      - Swift alert stream in Pub/Sub.
+        Messages contain the original alert bytes and metadata.
+
+BigQuery Tables
+^^^^^^^^^^^^^^^
+
+.. list-table::
+    :class: tight-table
+    :widths: 15 15 70
+    :header-rows: 1
+
+    * - Dataset
+      - Table
+      - Description
+
+    * - swift_alerts
```

Comment:
Just flagging to check this dataset name before merging (https://github.com/mwvgroup/Pitt-Google-Broker/pull/277/files#r2162347772).